src/cljc/harja/domain/oikeudet.cljc

```diff
@@ -96,6 +97,15 @@
       ;; Ei oikeutta
       nil)))
 
+;; Määritellään henkilötietoja sisältävät avaimet
+(def poistettavat-avaimet #{:etunimi, :sukunimi, :sahkoposti, :puhelin})
+
+;; Poistaa henkilötiedot ennen lokitusta.
+(defn- poista-henkilotiedot [kayttaja]
```

Comment:
:+1: 

Näkisin itse, että tää apufunktio sopis paremmin sinne datan alkulähteille, jossa se "käyttäja"-mapin sisältö varsinaisesti määritellään.
Tällöin apufunktiota voi tarvittaessa hyödyntää muissakin paikoissa, joissa tämä nimenomainen käyttäjä-mappi voidaan tällä apufunktiolla prosessoida.
Tuo "kayttaja" kulkee monessa muussakin paikassa funktioihin argumenteiksi, kuin pelkästään tässä oikeudet-nimiavaruudessa.

Kayttaja map määritellään nimiavaruudessa:
harja.palvelin.komponentit.todennus

funktiossa: varmista-kayttajatiedot

"poista-henkilotiedot" sopisi näiden joukkoon tuon funktion läheisyyteen.

Muissa nimiavaruuksissa voisi sitten tuota funktiota hyödyntää näin:
(todennus/poista-henkilotiedot kayttaja)

Reply:
Funktiokin voisi olla periaatteessa silloi kuvaavamman niminenkin:
Tyyliin,` (defn kayttaja-ilman-henkilotietoja [kayttaja] ....)`
Mutta, tästä voidaan kiistellä